README.md

```diff
@@ -37,6 +37,8 @@ If you don't know what version, start with the mbp and if it doesn't work ([Blan
 
 ## Installation
 
+**!! Warning: Versions newer than [5.7.19](https://github.com/marcosfad/mbp-ubuntu/releases/tag/v20.04-5.7.19-1) have various issues with init order and kernel modules and it's not recommended to use those**
```

Comment:
Perhaps the phrasing could be:
```
!! Warning: Versions newer than [5.7.19](https://github.com/marcosfad/mbp-ubuntu/releases/tag/v20.04-5.7.19-1) are considered in-development builds and **should not** be used due to issues from within the kernel and the init system.
```
Adds a bit more emphasis that they **really** shouldn't be used. But also hints that we haven't just totally dropped support.

Reply:
Yes thats fair enough